File: app/src/main/java/net/north101/android/ghplayertracker/data/CharacterPerk.java
package net.north101.android.ghplayertracker.data;

public class CharacterPerk {
    public final Perk perk;
    public int ticks;

    public CharacterPerk(Perk perk, int ticks) {
        this.perk = perk;
        this.ticks = ticks;
    }
}

File: app/src/main/java/net/north101/android/ghplayertracker/data/CharacterPerk.kt
package net.north101.android.ghplayertracker.data

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize
import net.north101.android.ghplayertracker.RecyclerItemCompare

@Parcelize
data class CharacterPerk(
    val perk: Perk,
    var ticks: Int
) : Parcelable, RecyclerItemCompare {
    override val compareItemId: String
        get() = perk.text
}
